docs/README.md

```diff
@@ -2541,7 +2528,42 @@ npm init
 npm install --save @boostercloud/framework-provider-aws-infrastructure
 ```
 
-The implementation of `InfrastructureRocket` might vary from one provider to the other, but in AWS it only requires two functions:
+The schema of an *Infrastructure Rocket* project should look something like this:
+
+```text
+rocket-your-rocket-name-aws-infrastructure
+├── package.json
+├── src
+    ├── lambdas
+    ├── index.ts
+    └── your-main-class.ts
+
+```
+
+In `<your-main-class>.ts` is where you define the two functions that AWS requires:
+
+```typescript
+export class YourMainClass {
+  public static mountStack(params: YourRocketParams, stack: Stack, config: BoosterConfig): void {
+    /* CDK code to expand your Booster infrastructure */
+  }
+  public static unmountStack?(params: YourRocketParams, utils: RocketUtils): void {
+    /* Optional code that runs before removing the stack */
+  }
+}
+```
+
+Let's look in more detail these two special functions:
+
+- **mountStack**: This function will run when you deploy your Booster application. Here you can use the [CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) code you need to extend the Booster functionality as you like. As we can see, it receives three params:
+  - `params`: The parameters required by your *Infrastructure Rocket* initializator, you will receive them from your Booster app's `config.ts` file.
+  - `stack`: An initialized AWS CDK stack that you can use to add new resources. Check out [the Stack API in the official CDK documentation](https://docs.aws.amazon.com/cdk/latest/guide/stacks.html#stack_api). This is the same stack instance that Booster uses to deploy its resources, so your resources will automatically be deployed along with the Booster's ones on the same stack. 
+  - `config`: It includes properties of the Booster project that is about to be deployed.
```

Comment:
- **mountStack**: Whenever we are deploying our Booster application (`boost deploy`) this method will also be run.  It receives three params:
   - `params`: The parameters required by your *Infrastructure Rocket* initializator, you will receive them from your Booster app's `config.ts` file.
   - `stack`: An initialized AWS CDK stack that you can use to add new resources. Check out [the Stack API in the official CDK documentation](https://docs.aws.amazon.com/cdk/latest/guide/stacks.html#stack_api). This is the same stack instance that Booster uses to deploy its resources, so your resources will automatically be deployed along with the Booster's ones on the same stack. 
   - `config`: It includes properties of the Booster project (e.g. project name) that come in handy for your rocket.